.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,107 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at:
+#
+#     https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+name: JKube Integration Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 1 * * *' # Everyday at 1
+
+env:
+  JKUBE_IT_REPOSITORY: https://github.com/jshiftio/jkube-integration-tests.git
+  JKUBE_IT_REVISION: master
+  JKUBE_IT_DIR: jkube-integration-tests
+
+jobs:
+  minikube:
+    name: Integration Tests in Minikube Cluster
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        minikube: [v1.5.1]
+        kubernetes: [v1.16.2,v1.12.0]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup Minikube-Kubernetes
+        uses: manusa/actions-setup-minikube@v1.0.0
```

Comment:
Could you please point where actions setting up Openshift and Minikube are?  Not sure if we should move it into jshift or not.

Reply:
https://github.com/marketplace/actions/setup-minikube-kubernetes-cluster

https://github.com/marketplace/actions/setup-okd-openshift-cluster